### PR TITLE
[MSPAINT] Update Portuguese (pt-PT) translation

### PR DESCRIPTION
**Pull request by zecarlos1957**
[MSPAINT] Update portuguese (pt-PT) translation

### CODE REVIEW

File: base/applications/mspaint/lang/pt-PT.rc [hunk 1]
```diff
@@ -223,23 +223,23 @@ BEGIN
...
-    IDS_ANGLE "The angle must be between -89 and 89."
-    IDS_LOADERRORTEXT "The file %s could not be loaded."
-    IDS_ENLARGEPROMPTTEXT "The image in the clipboard is larger than the bitmap.\nWould you like the bitmap enlarged?"
+    IDS_INTNUMBERS "Por favor, introduza apenas números inteiros!"
+    IDS_PERCENTAGE "A percentagem deve estar entre 1 e 500."
+    IDS_ANGLE "O ângulo deve estar entre -89 e 89."
+    IDS_LOADERRORTEXT "O ficheiro %s não pôde ser carregado."
+    IDS_ENLARGEPROMPTTEXT "A imagem na área de transferência é maior que o bitmap.\nDeseja ampliar o bitmap?"
     IDS_BOLD "Bold"
     IDS_ITALIC "Italic"
```
Comment (HBelusca) [inline base/applications/mspaint/lang/pt-PT.rc:241]: Translation?

File: base/applications/mspaint/lang/pt-PT.rc [hunk 2]
```diff
@@ -223,23 +223,23 @@ BEGIN
...
-    IDS_UNDERLINE "Underline"
+    IDS_UNDERLINE "Sublinhado"
     IDS_VERTICAL "Vertical"
```
Comment (HBelusca) [inline base/applications/mspaint/lang/pt-PT.rc:243]: Translation?

File: base/applications/mspaint/lang/pt-PT.rc [hunk 3]
```diff
@@ -223,23 +223,23 @@ BEGIN
...
     IDS_PRINTRES "%d x %d pixel/cm"
```
Comment (HBelusca) [inline base/applications/mspaint/lang/pt-PT.rc:244]: Does the unit need translation?

**Comment by HBelusca** [comment]
Cc @reactos/lang-portuguese

File: base/applications/mspaint/lang/pt-PT.rc [hunk 4]
```diff
@@ -6,7 +6,7 @@
  * TRANSLATOR: Manuel D V Silva
  */
 
-LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE
+LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL
```
[the remark this answers is not in the document]
Comment (zecarlos1957) [reply base/applications/mspaint/lang/pt-PT.rc:9]: As it says in https://reactos.org/wiki/Sublang

File: base/applications/mspaint/lang/pt-PT.rc (hunk 1 above)
Comment (zecarlos1957) [reply base/applications/mspaint/lang/pt-PT.rc]: Ok. Thanks.

File: base/applications/mspaint/lang/pt-PT.rc (hunk 2 above)
Comment (zecarlos1957) [reply base/applications/mspaint/lang/pt-PT.rc:243]: Vertical is equal in portuguese.

File: base/applications/mspaint/lang/pt-PT.rc (hunk 3 above)
Comment (zecarlos1957) [reply base/applications/mspaint/lang/pt-PT.rc:244]: pixel/cm is equal in portuguese.

File: base/applications/mspaint/lang/pt-PT.rc (hunk 4 above)
[the remark this answers is not in the document]
Comment (HBelusca) [reply base/applications/mspaint/lang/pt-PT.rc:9]: Note that there is also portuguese-brazilian translation in mspaint.

[the remark this answers is not in the document]
Comment (zecarlos1957) [reply base/applications/mspaint/lang/pt-PT.rc:9]: Yes, and they must use SUBLANG PORTUGUESE_BRAZILIAN.  Am i wrong?
In your perspective how should it be?

File: base/applications/mspaint/lang/pt-PT.rc [hunk 5]
```diff
@@ -19,19 +19,19 @@ BEGIN
         MENUITEM "&Guardar\tCtrl+G", IDM_FILESAVE
         MENUITEM "Guardar &como...", IDM_FILESAVEAS
         MENUITEM SEPARATOR
-        MENUITEM "From Scanner or Camera...", IDM_FILEFROMSCANNERORCAMERA
+        MENUITEM "Scanner ou Camera...", IDM_FILEFROMSCANNERORCAMERA
```
Comment (SecurityAndStuff) [inline base/applications/mspaint/lang/pt-PT.rc:22]:
```suggestion
        MENUITEM "Scanner ou câmera...", IDM_FILEFROMSCANNERORCAMERA
```

Comment (zecarlos1957) [reply base/applications/mspaint/lang/pt-PT.rc]: ops. Thanks.

**Review by HBelusca — APPROVED** [review]

**Review by binarymaster — APPROVED** [review]

**Pull request merged**